components/motor/roboclaw/roboclaw.go

```diff
@@ -4,21 +4,27 @@ package roboclaw
 
 import (
 	"context"
-	"errors"
 	"fmt"
 	"math"
 	"time"
 
 	"github.com/CPRT/roboclaw"
 	"github.com/edaniels/golog"
+	"github.com/pkg/errors"
 	utils "go.viam.com/utils"
 
 	"go.viam.com/rdk/components/motor"
 	"go.viam.com/rdk/operation"
 	"go.viam.com/rdk/resource"
 )
 
-var model = resource.DefaultModelFamily.WithModel("roboclaw")
+var (
+	model       = resource.DefaultModelFamily.WithModel("roboclaw")
+	connections map[string]*roboclaw.Roboclaw
+	baudRates   map[*roboclaw.Roboclaw]int
+)
+
+const maxRPM = 200
 
 // Config is used for converting motor config attributes.
 type Config struct {
```

Comment:
[q] Does it makes sense for this to be an always rebuild resource if any of these config parameters change? Ie, rebuild and recheck everything if any of these things change? I feel like the first four are definitely yes for that answer, but we should implement a `Reconfigure` and just update the structure and not rebuild for `TicksPerRotation` You can check out wheeled_base for an example of how the implement a Reconfigure with that logic. The `Reconfigure` method will need to be called out in the creator function.